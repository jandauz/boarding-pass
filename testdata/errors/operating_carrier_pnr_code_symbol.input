M1DESMARAIS/LUC       E!BC123 YULFRAAC 0834 326J001A0025 100