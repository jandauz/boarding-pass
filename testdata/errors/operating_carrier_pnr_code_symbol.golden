bcbp: Invalid data format:
  boarding pass data:
  | "M1DESMARAIS/LUC       E!BC123 YULFRAAC 0834 326J001A0025 100"
  |                         ^ got "!BC123 "
  |
  = reason: data for "Operating Carrier PNR Code" must be 7 alphanumeric characters with trailing whitespaces
